### ISSUE TITLE
[TW-959] Task N not found when using /old/new/

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#994]
_John Florian on 2011-01-17T13:23:44Z says:_ 

 I'm betting this is yet another parsing problem, but I cannot imagine how this might be interpreted in a way other than my intent:

<pre>
$ task info 99

Name                 Value                                                      
ID                   99
Description          Enable mayflower for media-playback systems.               
Status               Pending
Project              aos3.deploy                                                
This task blocked by 98 Create a mayflower package to effect upgrade to AOS 3.1.
UUID                 b3853c31-108e-1be9-2f0c-a3703e9b9724                       
Entered              1/17/2011 (10 mins)

$ task 99 /systems/systems in Mason/
Task 99 not found.
</pre>

Any ideas?

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2011-01-17T13:23:44Z
Modified: 2014-02-09T02:13:21Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2011-01-17T13:27:26Z says:_ 

 John, rerun like this, and we'll see what the parser did:

<pre>
task rc.debug:on 99 /systems/systems in Mason/
</pre>

I suspect the answer though will be quote that thing:

<pre>
task 99 "/systems/systems in Mason/"
</pre>

**Comment by taskwarrior** [comment]
_John Florian on 2011-01-17T13:37:54Z says:_ 

 Paul Beckingham wrote:
> John, rerun like this, and we'll see what the parser did:
> 
> [...]
> 
> I suspect the answer though will be quote that thing:
> 
> [...]


Funny, I was just reading #592 while trying to catch up on my mail and saw that advice there.  This gave me:

<pre>
Alias export -> export.yaml
Alias export.vcalendar -> export.ical
Alias ghistory -> ghistory.monthly
Alias history -> history.monthly
Alias rm -> delete
Timer Context::initialize 0.093710 sec
Hooks::initialize - hook system off
parse sequence '99'
parse cmd 'in'
parse description '/systems/systems Mason/'
auto filter: description.contains:/systems/systems
auto filter: description.contains:Mason/
Timer Context::parse 0.000208 sec
Timer TDB::loadPending 0.017447 sec
Timer TDB::loadPending 0.000172 sec
Timer TDB::commit 0.000000 sec
Timer Context::dispatch 0.023791 sec
Configuration override debug:on
Task 99 not found.
</pre>

While I'm not sure what the "in" command is (info maybe?), I did realize I needed to quote the thing.  Then I got you're reply.  Sorry for the noise.  I'm looking forward to a more strict syntax, even if it's dissimilar from my own "use style".
